adafruit_rfm69.py

```diff
@@ -143,7 +151,7 @@ def ticks_diff(ticks1, ticks2):
     return diff
 
 
-def check_timeout(flag, limit):
+def check_timeout(flag: int, limit: float) -> bool:
```

Comment:
`flag` is a function and not an int.
`limit` looks like it is an int.

Reply:
@gamblor21 check_timeout is called from line 808, where self.packet_sent and self.xmit_timeout are passed as arguments.  self.xmit_timeout is initialized as a float (2.0) and you are correct, packet_sent is a function that I will have to research.

Reply:
@gamblor21, @tekktrik so, packet_sent and payload_ready do indeed return an int, but they are testing a flag register and masking a particular bit, then shifting it to return a "1" or "0", so I am changing the return type from int to bool and likewise the flag in question to be a bool, which makes more sense for the name.  Thanks @gamblor21  for getting me to take a second look at these!